napari_cellseg3d/plugin_convert.py

```diff
@@ -33,29 +34,41 @@ def __init__(self, viewer: "napari.viewer.Viewer", parent):
         ########################
         # interface
 
-        self.btn_convert_folder_semantic = ui.make_button(
+        # label conversion
+        self.btn_convert_folder_semantic = ui.Button(
             "Convert to semantic labels", func=self.folder_to_semantic
         )
-        self.btn_convert_layer_semantic = ui.make_button(
+        self.btn_convert_layer_semantic = ui.Button(
             "Convert to semantic labels", func=self.layer_to_semantic
         )
-        self.btn_convert_folder_instance = ui.make_button(
+        self.btn_convert_folder_instance = ui.Button(
             "Convert to instance labels", func=self.folder_to_instance
         )
-        self.btn_convert_layer_instance = ui.make_button(
+        self.btn_convert_layer_instance = ui.Button(
             "Convert to instance labels", func=self.layer_to_instance
         )
-
-        self.btn_remove_small_folder = ui.make_button(
-            "Remove in folder", func=self.folder_remove_small
+        # remove small
+        self.btn_remove_small_folder = ui.Button(
+            "Remove small in folder", func=self.folder_remove_small
```

Comment:
Remove small cells ?

Reply:
It removes anything below a certain number of pixels in size, it works only for labels

Do you still have a cropped title ? It might be a Qt version/macOS specific issue